### ISSUE TITLE
Arrangement of vital signs on workup page is sub-optimal

### ISSUE THREAD

**Issue by jashephe** [SaturdayNeighborhoodHealthClinic/osler#246]
The arrangement of the vital signs on the workup page is not optimal. The systolic blood pressure, for instance, is on the opposite side of the page and on a different line from the diastolic pressure. These could be rearranged to speed provider entry. I would encourage to use:

```
--------------------------------------------------
| Systolic BP   | Height      | Temperature      |
--------------------------------------------------
| Diastolic BP  | Weight      | Respiratory Rate |
--------------------------------------------------
| Heart Rate    |             |                  |
--------------------------------------------------
```
This has the benefit of keeping BP and HR in the order that they are read on the the automated blood pressure cuffs in clinic.

**Comment by justinrporter** [comment]
Potentially fixed in #239 -- @newboldd ?

**Comment by justinrporter** [comment]
Fixed in #239
